addons/im_livechat/tests/test_user_livechat_username.py

```diff
@@ -81,3 +81,34 @@ def test_user_livechat_username_reactions(self):
                 "write_date": fields.Datetime.to_string(john.partner_id.write_date),
             },
         )
+
+    def test_user_livechat_username_in_store(self):
+        dog_expert = self.env["im_livechat.expertise"].create({"name": "dog"})
+        john = self._create_operator("fr_FR", expertises=dog_expert)
+        operator = self._create_operator("en_US")
+
```

Comment:
I usually prefer not having unnecessary empty lines in functions, either they are useless, or they indicate there is a missing comment, or a function that should be split into smaller one.